ci/run_playbooks.yml

```diff
@@ -39,7 +39,7 @@
         - name: Patch the openstackversions to use the master containers for aodh and heat
           ansible.builtin.shell:
             cmd: |
-              oc patch openstackversions openstack-galera-network-isolation --type merge --patch-file ci/patch-openstack-versions.yaml
+              oc patch  oscp/controlplane  --type merge --patch-file ci/patch-openstack-versions.yaml
```

Comment:
This is originally a patch to the openstackversions CR, not to the OpenStackControlePlane.

Has openstackversions been removed?
It probably has been renamed, which should be addressed, but please doublecheck the resource type

Reply:
my mistake thanks emma I updated it